polkadot/node/subsystem-util/src/vstaging.rs

```diff
@@ -35,7 +35,7 @@ const LOG_TARGET: &'static str = "parachain::subsystem-util-vstaging";
 pub async fn get_disabled_validators_with_fallback<Sender: SubsystemSender<RuntimeApiMessage>>(
 	sender: &mut Sender,
 	relay_parent: Hash,
-) -> Result<Vec<ValidatorIndex>, Error> {
+) -> Result<Vec<ValidatorIndex>, runtime::Error> {
```

Comment:
Just asking for my own benefit :)
We prefer `runtime::Error` here since `get_disabled_validators_with_fallback` wraps a runtime call and it is more natural or I am missing something more important?

Reply:
It is to satisfy the interface in RuntimeInfo::get_disabled_validators